client/src/components/landing/Landing.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { Card } from 'antd';
+
+import LoginForm from './LoginForm';
+import RegisterForm from './RegisterForm';
+import './Landing.css';
+
+const Landing = props => {
+  const tabList = [
+    { key: 'login', tab: 'Login' },
+    { key: 'register', tab: 'Register' },
+  ];
+
+  const [tabKey, setTabKey] = useState('login');
```

Comment:
What does this do?

Reply:
`useState` is a react hook that allows us to create and manage state local to the component. It returns the value of the state itself (`tabKey`) and a function used to update the state (`setTabKey`). The arg passed to `useState` is the initial/default state. This particular instance is setting up some state to track which tab we're on (login or register).